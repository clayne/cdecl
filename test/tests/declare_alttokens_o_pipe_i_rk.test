cdecl @ /dev/null @ -a -xc++ @ define S as struct S; declare | as operator (x as int) returning reference to S @ 0
