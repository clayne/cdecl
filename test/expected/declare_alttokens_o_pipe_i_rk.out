S bitand operator bitor(int x);
